.github/workflows/util/execute_and_retry.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This function is for retrying commands in the case they fail. It accepts three arguments
+# $1: Number of retries it will attempt
+# $2: Command to execute
+# $3: Clean up commands
+execute_and_retry () {
+  retry_counter=0
+  max_retry=$1
```

Comment:
Can we make it more clear and update the script accordingly?
```suggestion
  max_retry=$1
  command=$2
  cleanup=$3
```